integrationTest/test/webpack-zero-conf-karma/.gitignore

```diff
@@ -0,0 +1,27 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Is this `.gitignore` file really needed? We have a pretty good gitignore file at the root, no?

Reply:
Does it filter the dist folder?